test/meta-colony.js

```diff
@@ -172,10 +190,10 @@ contract("Meta Colony", accounts => {
       const skill3ParentSkillId2 = await colonyNetwork.getParentSkillId(6, 1);
       expect(skill3ParentSkillId2).to.eq.BN(1);
 
-      const skill4 = await colonyNetwork.getSkill(7);
+      const skill4 = await colonyNetwork.getSkill(2);
```

Comment:
What's wrong with testing skillId 7? Skill 2 is the reputation mining skill which is probably tested elsewhere but [here](https://github.com/JoinColony/colonyNetwork/pull/567/files#diff-07c4d8d59a6ece14a120308fcc8f0e1bR147) we are adding the new global skill with Id 7 so it should be that which is tested.

Reply:
I was trying to preempt this comment with [this explanation](https://github.com/JoinColony/colonyNetwork/pull/567/files/985928455386ede5568c68b4347b49610b297828#diff-07c4d8d59a6ece14a120308fcc8f0e1bR141).

Reply:
The explanation link is now gone..

Reply:
Let's [try again](https://github.com/JoinColony/colonyNetwork/pull/567/files#diff-07c4d8d59a6ece14a120308fcc8f0e1bR146)...